src/bp/ui-studio/src/web/actions/flows-api/api.ts

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios'
+
+export const deleteFlow = async (flowName: string) => {
+  flowName = escapeForwardSlashes(flowName)
+  await axios.delete(`${window.BOT_API_PATH}/flow/${flowName}`)
+}
+
+export const insertFlow = async flow => {
+  await axios.post(`${window.BOT_API_PATH}/flow`, { flow })
+}
+
+export const updateFlow = async (flowName: string, flow) => {
+  flowName = escapeForwardSlashes(flowName)
+  await axios.put(`${window.BOT_API_PATH}/flow/${flowName}`, { flow })
+}
+
+const escapeForwardSlashes = (pathParam: string) => {
+  return pathParam.replace(/\//g, '%2F')
+}
```

Comment:
If we are to move api calls (axios) outside actions.js, I suggest creating a generic api.ts file which will include ALL of the api calls. Otherwise we will always have to dig to find something